src/cosalib/build.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, *args, **kwargs):
 
         self._found_files = {}
         self._workdir = kwargs.pop("workdir", os.getcwd())
-        self._tmpdir = tempfile.mkdtemp(prefix="build_tmpd")
+        self._tmpdir = tempfile.mkdtemp(prefix="build_tmpd", dir=self._workdir)
```

Comment:
This hunk can be dropped now, it's conflicting.

Reply:
:100: I added that to get it to pass CI. Rebasing.

Reply:
Dropped and rebased. Running through CI now.